web/packages/shared/components/AccessRequests/NewRequest/useSpecifiableFields.ts

```diff
@@ -78,6 +78,9 @@ export function useSpecifiableFields() {
    */
   const [pendingRequestTtl, setPendingRequestTtl] = useState<Option<number>>();
 
+  const [reasonMode, setReasonMode] = useState<string>();
+  const [reasonPrompts, setReasonPrompts] = useState<string[]>();
```

Comment:
i don't think we need states for these, since these are static data right? (normally useState is used if the values can change)

we already save the dry run response as `dryRunResponse`, so we can access these static fields there

Reply:
I monkey-coded that with my limited frontend knowledge. I'd be happy to improve that if you can tell me how to do that. I failed to somehow obtain that directly even though that that's the most reasonable thing to do.

Reply:
That works. Thank you!